src/PhpBrew/VariantBuilder.php

```diff
@@ -561,6 +562,39 @@ public function __construct()
             }
         };
 
+        /*
+         * --with-sodium option
```

Comment:
This is an implementation detail that may change for a newer PHP version. Please remove.